scripts/generators/es_template.py

```diff
@@ -59,6 +59,11 @@ def entry_for(field):
             ecs_helpers.dict_copy_existing_keys(field, field_entry, ['ignore_above'])
         elif field['type'] == 'text':
             ecs_helpers.dict_copy_existing_keys(field, field_entry, ['norms'])
+        elif field['type'] == 'alias':
```

Comment:
Similar to the `beats` generator, since we're checking earlier that `path` is present when `type: alias` in `schema.cleaner.field_mandatory_attributes`, I think we're ok to just copy without worrying about an exception.

```python
       elif field['type'] == 'alias':
            ecs_helpers.dict_copy_existing_keys(field, field_entry, ['path'])
```

Reply:
Fixed in https://github.com/elastic/ecs/pull/877/commits/231fe9870a9783c5053bfaa29ad6187a32e201f0